types/command-line-args/index.d.ts

```diff
@@ -1,87 +1,86 @@
-// Type definitions for command-line-args 4.0.7
+// Type definitions for command-line-args 5.0.2
 // Project: https://github.com/75lb/command-line-args
-// Definitions by: CzBuCHi <https://github.com/CzBuCHi>
+// Definitions by: CzBuCHi <https://github.com/CzBuCHi>, Lloyd Brookes <https://github.com/75lb>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.2
 
 /**
- * Returns an object containing all options set on the command line. By default it parses the global  [`process.argv`](https://nodejs.org/api/process.html#process_process_argv) array.
- *
- * By default, an exception is thrown if the user sets an unknown option (one without a valid [definition](#exp_module_definition--OptionDefinition)). To enable __partial parsing__, invoke `commandLineArgs` with the `partial` option - all unknown arguments will be returned in the `_unknown` property.
- *
- *
- * @param {module:definition[]} - An array of [OptionDefinition](#exp_module_definition--OptionDefinition) objects
- * @param [options] {object} - Options.
- * @param [options.argv] {string[]} - An array of strings, which if passed will be parsed instead  of `process.argv`.
- * @param [options.partial] {boolean} - If `true`, an array of unknown arguments is returned in the `_unknown` property of the output.
- * @returns {object}
- * @throws `UNKNOWN_OPTION` if `options.partial` is false and the user set an undefined option
- * @throws `NAME_MISSING` if an option definition is missing the required `name` property
- * @throws `INVALID_TYPE` if an option definition has a `type` value that's not a function
- * @throws `INVALID_ALIAS` if an alias is numeric, a hyphen or a length other than 1
- * @throws `DUPLICATE_NAME` if an option definition name was used more than once
- * @throws `DUPLICATE_ALIAS` if an option definition alias was used more than once
- * @throws `DUPLICATE_DEFAULT_OPTION` if more than one option definition has `defaultOption: true`
- * @alias module:command-line-args
+ * Returns an object containing option values parsed from the command line. By default it parses the global `process.argv` array. Parsing is strict by default. To be more permissive, enable `partial` or `stopAtFirstUnknown` modes.
```

Comment:
Why removed information about params and throws?

Reply:
Because those exceptions no longer exist in version 5.